src/frontend/src/components/ChangeRequestRow.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Typography, Grid, useTheme } from '@mui/material';
+import { Box } from '@mui/system';
+import { ChangeRequest } from 'shared/src/types/change-request-types';
+import ChangeRequestDetailCard from './ChangeRequestDetailCard';
+
+interface ChangeRequestRowProps {
+  title: string;
+  changeRequests: ChangeRequest[];
+  noChangeRequestsMessage: string;
+}
+
+const ChangeRequestRow: React.FC<ChangeRequestRowProps> = ({ title, changeRequests, noChangeRequestsMessage }) => {
+  const theme = useTheme();
+
+  const displayCRCards = (crList: ChangeRequest[]) => (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'row',
+        flexWrap: 'wrap',
+        overflow: 'auto',
+        justifyContent: 'flex-start',
+        '&::-webkit-scrollbar': {
+          height: '20px'
+        },
+        '&::-webkit-scrollbar-track': {
+          backgroundColor: 'transparent'
+        },
+        '&::-webkit-scrollbar-thumb': {
+          backgroundColor: theme.palette.divider,
+          borderRadius: '20px',
+          border: '6px solid transparent',
+          backgroundClip: 'content-box'
+        }
+      }}
+    >
+      {crList.map((cr: ChangeRequest) => (
+        <ChangeRequestDetailCard changeRequest={cr}></ChangeRequestDetailCard>
+      ))}
+    </Box>
+  );
+
+  const renderChangeRequests = (title: string, crList: ChangeRequest[], emptyMessage: string) => {
```

Comment:
since this is only called in one place just put all this code where it's being called below